tests/unit/amo/components/TestAddon.js

```diff
@@ -781,11 +781,77 @@ describe(__filename, () => {
       }),
     });
 
-    expect(root.find('.AddonDescription')).toHaveLength(1);
+    expect(root.find('.AddonDescription')).toHaveLength(0);
+  });
 
-    expect(root.find('.AddonDescription-contents')).toHaveHTML(
-      `<div class="AddonDescription-contents">${summary}</div>`,
-    );
+  it('does not display anything when the lightweight theme has no description', () => {
+    const summary = 'my theme is very cool';
+    const root = shallowRender({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_THEME,
+        summary,
+        description: null,
+      }),
+    });
+
+    expect(root.find('.AddonDescription')).toHaveLength(0);
+  });
+
+  it('does not display anything when the generic add-on has no description', () => {
+    const summary = 'my theme is very cool';
+    const root = shallowRender({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        type: ADDON_TYPE_COMPLETE_THEME,
```

Comment:
hmm..I have never really seen this type before : /

We can just update the description so it matches: "..when the complete theme has no description"?